internal/clientgen/clientgentypes/clientgentypes.go

```diff
@@ -63,3 +64,53 @@ func NewServiceSet(md *meta.Data, include, exclude []string) ServiceSet {
 func AllServices(md *meta.Data) ServiceSet {
 	return NewServiceSet(md, []string{"*"}, nil)
 }
+
+type TagSet struct {
+	includedTags []string
+	excludedTags []string
```

Comment:
Make these `map[string]bool` to avoid running into O(N^2) performance in the `IsRPCIncluded` method

Reply:
Done